README.md

```diff
@@ -44,6 +44,7 @@ The following HTTP headers will be expected:
 ------------------|----------------------------------|-------------------------------------|
  Content-Type     | Content type of the request body | Always has to be `application/json` |
 
+The request body size must not exceed 10 MB.
```

Comment:
Ich würde diesen Satz hier weglassen und genau so in die Description der Tabelle packen. Ich vermute, dass so Übersichtstabellen noch am ehesten gelesen werden. Was meinst du?

Reply:
Mein Eindruck ist, dass das unten nur kurze Hinweise sind. Deshalb hier oben die genaue Zahl.